### DISCUSSION TITLE
Apk builds with firebase_crashlytics_collection_enabled = false

### DISCUSSION THREAD

**Discussion by sbatikeri** [invertase/react-native-firebase#4251] [Q&A] [answered]
Recently I have implemented firebase in our project. I am continuously  having multiple issues related to crashlytics. I am not constantly seeing crashes / non-fatal on dashboard. I use to see non-fatal errors for api failures but not now. After looking at the recent issues I have added following settings to firebase.json file. I did notice that in my apk manifest file I did see  firebase_crashlytics_collection_enabled by default false but I see the crashes on dashboard. Can some tell me when will firebase_crashlytics_collection_enabled set to true. 

{
    "react-native": {
        "crashlytics_disable_auto_disabler": true,
        "crashlytics_debug_enabled": true,
        "crashlytics_auto_collection_enabled": true,
        "perf_auto_collection_enabled": true,
        "analytics_auto_collection_enabled": true
    }
} 
<img width="873" alt="Screen Shot 2020-09-11 at 3 25 53 PM" src="https://user-images.githubusercontent.com/62160090/92969767-3062f800-f443-11ea-944a-5245f6694850.png">

**Comment by mikehardy** [comment, marked as answer]
Seems like more of an "issue" than a question really, I'd want to see the  details requested in our issue template

**Reply by sbatikeri** [reply]
ok, I will create issue.
